man/coredump.conf.xml

```diff
@@ -109,6 +109,21 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AccessContainer=</varname></term>
```

Comment:
AccessNamepsace instead? Honestly, find "Follow" prefix quite misleading too.

Reply:
Btw, if you look at the documentation it explains how this works so I think that even AccessContainer= could stay because after all user configuration should be as high-level as practical and users (unless they are debugging/testing or so) run containers not individual namespaces.

Reply:
nah, people use all kind of stuff. for example, systemd's namespacing logic for services is not at all something we call "containers". Please rename this.

Reply:
So Follow... then?